portal/component-tests/fixtures/deal-fully-completed.js

```diff
@@ -136,8 +136,41 @@ const deal = {
         status: 'Completed',
       },
       {
-        _id: '789123',
+        _id: '6597dffeb5ef5ff4267e5045',
         facilityStage: 'Unconditional',
+        hasBeenIssued: false,
+        'requestedCoverStartDate-day': `${format(now, 'dd')}`,
+        'requestedCoverStartDate-month': `${format(now, 'MM')}`,
+        'requestedCoverStartDate-year': `${format(now, 'yyyy')}`,
+        'coverEndDate-day': `${format(nextMonth, 'dd')}`,
```

Comment:
any reason `requestedCoverStart...` and `coverEnd...` aren't named consistently?

Reply:
This is a historic structure to accomodate for different deal types.